samples/notebooks/dotnet/09-BingSearch-using-kernel.ipynb

```diff
@@ -0,0 +1,224 @@
+     "output_type": "stream",
+     "text": [
+      "Example 1\n",
+      "What is quantum tunnelling\n",
+      "----\n",
+      "In physics, quantum tunnelling, barrier penetration, or simply tunnelling is a quantum mechanical phenomenon in which an object such as an electron or atom passes through a potential energy barrier that, according to classical mechanics, the object does not have sufficient energy to enter or surmount.\n",
+      "\n",
+      "Example 2\n",
+      "[1] Elon Musk (@elonmusk) is the owner of Twitter and is the most-followed person on Twitter, with over 144 million followers. Former U.S. President Barack Obama (@BarackObama) is the second most-followed person on Twitter, with over 132 million followers.\n"
+     ]
+    }
+   ],
+   "source": [
+    "// Load Bing skill\n",
+    "//Uncomment the Example \n",
+    "using (var bingConnector = new BingConnector(\"BING_KEY\")\n",
+    "{\n",
+    "kernel.ImportSkill(new WebSearchEngineSkill(bingConnector), \"bing\");\n",
+    "//First run the example sample given below and uncomment the await the  following statements \n",
+    "//await Example1Async(kernel);\n",
+    "//await Example2Async(kernel);\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "private static async Task Example1Async(IKernel kernel)\n",
+    "{\n",
+    "        Console.WriteLine(\"Example 1\");\n",
+    "\n",
+    "        // Run \n",
+    "        var question = \"What is quantum tunnelling\";\n",
+    "        var bingResult = await kernel.Func(\"bing\", \"search\").InvokeAsync(question);\n",
+    "\n",
+    "        Console.WriteLine(question);\n",
+    "        Console.WriteLine(\"----\");\n",
+    "        Console.WriteLine(bingResult);\n",
+    "        Console.WriteLine();\n",
+    "\n",
+    "        /* OUTPUT:\n",
+    "\n",
+    "            What is quantum tunnerling\n",
+    "            ----\n",
+    "            In physics, quantum tunnelling, barrier penetration, or simply tunnelling is a quantum mechanical\n",
+    "            phenomenon in which an object such as an electron or atom passes through a potential energy \n",
+    "            barrier that, according to classical mechanics, the object does not have sufficient energy to\n",
+    "            enter or surmount.\n",
+    "            \n",
+    "       */\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "dotnet_interactive": {
+     "language": "csharp"
+    },
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "private static async Task Example2Async(IKernel kernel)\n",
+    "{\n",
+    "    Console.WriteLine(\"Example 2\");\n",
+    "    //Use the following function in VS only\n",
```

Comment:
Not sure I understand this comment... this should work anywhere the user is running the notebook. Why VS only? (Seems to work fine in VSCode)

Reply:
My mistake it should be 
`//The following function will only works in Polygot and Jupyter Notebook`